.goreleaser.yml

```diff
@@ -100,3 +100,18 @@ snapcrafts:
         command: nginx-prometheus-exporter
         plugs: ["network", "network-bind"]
     disable: "{{ if .IsSnapshot }}true{{ end }}"
+
+nix:
+  - repository:
+      owner: nginxinc
+      name: nur
```

Comment:
is this a typo?

Reply:
`nur`? No, it stands for `Nix User Repository`